### DISCUSSION TITLE
How can I set the maximum/minimum height of Milkdown

### DISCUSSION THREAD

**Discussion by SimonShiki** [Milkdown/milkdown#447] [Q&A] [answered]
I tried to use a ``<div>`` element on the outer layer of the component and set its maximum height to make it to stay fixed in size, but it doesn't seem to work.

**Comment by SimonShiki** [comment]
I find the answer, thanks

**Comment by Saul-Mirone** [comment]
Seems it's a css question?

**Reply by SimonShiki** [reply]
yep, I refer to the demo and set ``height`` instead of ``max-height`` to solve this problem.

What's more, As a component,  It does not seem to be friendly to the need for personalized css modification( Eg: module.css in Next.js). Is there a convenient way to do it?

**Reply by Saul-Mirone** [reply, marked as answer]
In v6.0 we provide a way for users to create custom theme much more easier.

**Reply by SimonShiki** [reply]
> In v6.0 we provide a way for users to create custom theme much more easier.

okay, thanks a lot!
